src/redux/reducer.js

```diff
@@ -0,0 +1,16 @@
+import { SET_LOADING } from '../utils/constants';
```

Comment:
Remove Redux. We do not need it in the application

Reply:
Removed